src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -773,8 +773,12 @@ int fx_muxer_t::read_config_and_execute(
     return execute_app(impl_dir, &init, new_argc, new_argv);
 }
 
-/* static */
-int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
+/**
+ *  Main entrypoint to detect operating mode and perform corehost, muxer,
```

Comment:
Do we still have anything called corehost? I would expect the comment to only refer to "shared and standalone host" modes?